fluentgen/readme.md

```diff
@@ -0,0 +1,30 @@
+
+#### Javagen
+
+``` yaml
+use-extension:
+  "@autorest/modelerfour": "4.6.200"
+
+pipeline:
+
+  modelerfour:
+    flatten-models: true
+    flatten-payloads: true
+    naming:
+      preserve-uppercase-max-length: 4
+
+  fluentgen:
+    scope: java
```

Comment:
Should scope be `fluent`?

Reply:
Sure. I will change it if it works. My concern is I do not know how autorest works on the scope, and if I change it to fluent, whether it will load README.java.md, if it is fluent scope.

Reply:
Tried but I cannot change it to `fluent` without at least changing all README.md in swagger spec. 
Getting
`[Exception] No input files provided.`

These READMEs are all tagged with java
https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/resources/resource-manager/readme.java.md

Reply:
Ok we can use java for now. 
My concern is in AutoRest there's a default codegen lookup. For example, `--java` maps to a default npm package `autorest.java`. My understanding is that fluent will be shipped as a separate npm package. 
I think something like `--java --fluent` may be fitted for the fluent gen where `--fluent` can be used to distinguish the javagen from fluentgen while `--java` will enable autorest to pick up README.java.md.

Reply:
Yes. This is a real problem.

I will follow with Garrett to see that, under same package and same scope=java, whether we can config in readme.md to have a switch --fluent to switch the pipeline. I.e. no --fluent -> preprocesser -> javagen, with --fluent -> fluentnamer -> fluentgen